src/api/converter.ts

```diff
@@ -521,9 +521,9 @@ function fromTimeTicket(pbTimeTicket: PbTimeTicket): TimeTicket {
   }
 
   return TimeTicket.of(
-    Long.fromString(pbTimeTicket.getLamport(), true),
+    Long.fromString(`${pbTimeTicket.getLamport()}`, true),
```

Comment:
It's a little weird why it has to be changed.

Reply:
Could you please share the command you used to build `protobuf`?

I'm not sure about the details of 'protobuf', but looking at the code of the currently built'yorkie_pb.js' it seems that the build went differently.

```js
// npm run build:proto
/**
 * Serializes the given message to binary data (in protobuf wire
 * format), writing to the given BinaryWriter.
 * @param {!proto.api.TimeTicket} message
 * @param {!jspb.BinaryWriter} writer
 * @suppress {unusedLocalVariables} f is only used for nested messages
 */
proto.api.TimeTicket.serializeBinaryToWriter = function(message, writer) {
  var f = undefined;
  f = message.getLamport();
  if (parseInt(f, 10) !== 0) {
    writer.writeUint64String(
      1,
      f
    );
  }
  
  ...
};
```

```js
// this PR
/**
 * Serializes the given message to binary data (in protobuf wire
 * format), writing to the given BinaryWriter.
 * @param {!proto.api.TimeTicket} message
 * @param {!jspb.BinaryWriter} writer
 */
proto.api.TimeTicket.serializeBinaryToWriter = function(message, writer) {
  var f = undefined;
  f = message.getLamport();
  if (f !== 0) {
    writer.writeUint64(
      1,
      f
    );
  }

  ...
};
```

Inferring through the converted code, the converted code currently used in PR treats Uint64 as a number.

<img width="629" alt="Screen Shot 2021-02-03 at 9 24 44 PM" src="https://user-images.githubusercontent.com/40932237/106747577-707f7d00-6667-11eb-8cc5-305f84e6e4ce.png">

---

You can build `protobuf` through the `npm run build:proto` command declared in the project.
On my local, I ran the `protobuf` build again and saw improvement.

Can you test it locally and tell us the results?

Reply:
@dc7303 
Oh sorry, I used oldest version of protobuf(3.2.0). Now it works after changing version to latest(3.14.0). I just pushed with a commit.